lib/ui/annotations.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// TODO(dnfield): Remove unused_import ignores when https://github.com/dart-lang/sdk/issues/35164 is resolved.
+
+// @dart = 2.6
+part of dart.ui;
+
+/// Annotation used by Flutter's Dart compiler to indicate that an
```

Comment:
Should the short description reference what happens in release vs. debug?

Reply:
The way the flag is implemented, it's not really specific to release or debug.

When we get to a point where we'd turn this on by default in release we should probably mention something.  I'll add a TODO to update the docs for this when that happens.